modules/server/main.tf

```diff
@@ -33,8 +33,8 @@ module "icd_dictionary" {
 
 module "mdr" {
   source         = "./mdr"
-  mdr_version    = "4.0.4-SNAPSHOT"
-  mdr-ui_version = "2.0.6-SNAPSHOT"
+  mdr_version    = "release-gba-mdr"
```

Comment:
I don't like to deploy floating releases. Some kind of version would be highly appreciated.

Reply:
We decided to take the risk.